PSSwagger/Utilities.psm1

```diff
@@ -204,4 +204,28 @@ function Get-ParameterGroupName {
             return "HyphenMinus$($OperationId)HyphenMinus$Postfix"
         }
     }
+}
+
+function Get-FormattedFunctionContent {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory=$false)]
+        [string]
+        $Formatter,
+
+        [Parameter(Mandatory=$false)]
+        [string[]]
+        $Content
+    )
+
+    $singleStringContent = $Content | Out-String
+    $singleStringContent = $singleStringContent -replace "`r`n","`n"
+    $singleStringContent = $singleStringContent -replace "`r","`n"
+    if ($Formatter -eq 'None') {
+        $singleStringContent
+    } elseif ($Formatter -eq 'PSScriptAnalyzer') {
+        PSScriptAnalyzer\Invoke-Formatter -ScriptDefinition $singleStringContent
+    } else {
+        $singleStringContent
+    }
```

Comment:
Minor: Above if($Formatter -eq 'None') {} is not required as else part will handle the that case as well.

Reply:
This is the default case in the event a dev forgets to include a new option here. Shouldn't ever be hit, but can be so is included for safety.